.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: build
+name: CI
```

Comment:
Let's do it in separate ticket. Because there are other improvements that we can apply here. For example we can skip pmm-admin tests if only pmm-agent code was modified.

I will create ticket for that